pylintrc

```diff
@@ -78,7 +78,7 @@ enable=
 # --disable=W"
 
 # These are errors still not addressed in PyWren.
-disable=invalid-name,missing-docstring,too-many-branches,too-many-arguments,too-many-locals,logging-format-interpolation,too-many-public-methods,too-few-public-methods,too-many-instance-attributes,bare-except,broad-except,protected-access,unidiomatic-typecheck,too-many-return-statements,too-many-statements,locally-disabled,len-as-condition,no-else-return,superfluous-parens,inconsistent-return-statements,logging-not-lazy,useless-import-alias
+disable=invalid-name,missing-docstring,too-many-branches,too-many-arguments,too-many-locals,logging-format-interpolation,too-many-public-methods,too-few-public-methods,too-many-instance-attributes,bare-except,broad-except,protected-access,unidiomatic-typecheck,too-many-return-statements,too-many-statements,locally-disabled,len-as-condition,no-else-return,superfluous-parens,inconsistent-return-statements,logging-not-lazy,useless-import-alias,old-style-class,bad-option-value,useless-object-inheritance
```

Comment:
what is "bad-option-value"  ?

Reply:
On my local pylint version this line:
https://github.com/pywren/pywren/pull/265/commits/cafdef6fae5260b5ca7d863132311f688369cd69#diff-1fc418d69548b6c23ddaa8244d679115R321
causes an error.

I didn't want to semantically change this piece of code. So I added a an ignore. But then the travis version of pylint (on python2) didn't recognize this flag  and threw another error.  So I just told pylint to ignore this error. Does that make sense?